cmd/flags_core.go

```diff
@@ -25,6 +26,12 @@ func CoreFlags() *flag.FlagSet {
 		"Indicates node to connect to the given remote core node. "+
 			"Example: <protocol>://<ip>:<port>, tcp://127.0.0.1:26657",
 	)
+	flags.String(
+		coreGRPCFlag,
+		"",
+		"Indicates node to connect to the given core node gRPC endpoint for state-related queries"+
```

Comment:
As this is user-facing we should be even more clear which gRPC endpoints we are talking about. It's the app not core that exposes these.